en/mkdocs.yml

```diff
@@ -69,26 +69,25 @@ nav:
     - 'Configuring Users and Roles': install-and-setup/configuring-users-and-roles.md
     - 'Configuring SMS OTP Authenticator': install-and-setup/configuring-sms-otp-authenticator.md
   - 'Learn':
-    - 'Consent Management':
-      - 'Introduction': learn/consent-management.md
-      - 'Consent Amendment History': learn/consent-amendment-history.md
+    - 'Consent Management': learn/consent-management.md
     - 'Consent Authorization':
        - 'Introduction': learn/consent-authorization-intro.md
        - 'Pushed Authorization Endpoint': learn/pushed-authorization-endpoint.md
     - 'Token Authentication': learn/token-authentication.md
     - 'Dynamic Client Registration':
        - 'Introduction': learn/dynamic-client-registration.md
-       - 'Configuration': learn/dynamic-client-registration-configuration.md
+       - 'Configuration': learn/custom-dcr-validator.md
     - 'Consumer Authentication': learn/consumer-authentication.md
     - 'Metadata Cache Management': learn/metadata-cache-management.md
     - 'Administration API': learn/administration-api.md
     - 'Arrangement Revocation API': learn/arrangement-revocation-api.md
     - 'API Security':
-        - 'Introduction': learn/api-security.md
-        - 'Configuration': learn/api-security-configurations.md
+       - 'Introduction': learn/api-security.md
+       - 'Configuration': learn/api-security-configuration.md
```

Comment:
Rename the page to 'configurations' for consistency with other spaces.

Reply:
in other pages it's 'configuration'

Reply:
oh, apologies!! let's have it as it is then.